code/mob.dm

```diff
@@ -1646,11 +1647,11 @@
 	else
 		src.color_matrices -= label
 
-	src.update_active_matrix()
+	src.update_active_matrix(respect_view_tint_settings)
 
 /// Multiplies all of the mob's color matrices together and puts the result into src.active_color_matrix
 /// This matrix will be applied to the mob at the end of this proc, and any time the client logs in
-/mob/proc/update_active_matrix()
+/mob/proc/update_active_matrix(respect_view_tint_settings)
```

Comment:
This argument doesn't appear to be used?

Reply:
It's used in line 1670 right now, or 1669 with your proposed changes